DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>I got a negative ID in my run. Did we ever figure out why there were some negative IDs? Maybe we need to explain this in case some people retrieve negative IDs  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>I think this is int64 thing and we don't need to explain in this particular NB?</p>

Reply:
Ok! Whatever you think is best, I just flagged because I remember people asking about this but its probably fine.